config/locales/en.yml

```diff
@@ -662,14 +668,10 @@ en:
     request_amends:
       heading: Request amends
     respond:
-      heading: Marking it as sent
-      reminders_html: |
-        <strong>Make sure you have:</strong>
-        <ul class="list list-bullet">
-          <li>cleared the response with the Deputy Director</li>
-          <li>uploaded the response and any supporting documents</li>
-          <li>sent the response to the person who made the request</li>
-        </ul>
+      heading: Mark as sent
+      respond_date: Add date this response was sent to ICO
```

Comment:
this copy is appearing for both FOI and SAR
![image](https://user-images.githubusercontent.com/3441519/43710050-69650d6c-9966-11e8-9e78-fb1700a4ae5e.png)